[16:44:04] [DEBUG] Running client sanity checks...
[16:44:04] [DEBUG] Loaded Extension: interactions.ext.jurigged
[16:44:06] [DEBUG] No setup function found in play
[16:44:06] [DEBUG] Found extension class Music in play: Attempting to load
[16:44:06] [DEBUG] Added callback: Music._menu
[16:44:06] [DEBUG] Added callback: Music._pause
[16:44:06] [DEBUG] Added callback: Music._resume
[16:44:06] [DEBUG] Added callback: Music._skip
[16:44:06] [DEBUG] Added callback: Music._stop
[16:44:06] [DEBUG] Added callback: Music.play
[16:44:06] [DEBUG] Loaded Extension: play
[16:44:06] [DEBUG] No setup function found in askbard
[16:44:06] [DEBUG] Found extension class Bard in askbard: Attempting to load
[16:44:06] [DEBUG] Added callback: Bard._askbard
[16:44:06] [DEBUG] Added callback: Bard._endbard
[16:44:06] [DEBUG] Added callback: Bard.set_bard_channel
[16:44:06] [DEBUG] Loaded Extension: askbard
[16:44:06] [DEBUG] No setup function found in noi_chu
[16:44:06] [DEBUG] Found extension class NoiChu in noi_chu: Attempting to load
[16:44:06] [DEBUG] Added callback: NoiChu._reset
[16:44:06] [DEBUG] Added callback: NoiChu._world_setup
[16:44:06] [DEBUG] Loaded Extension: noi_chu
[16:44:06] [DEBUG] No setup function found in db_refesh
[16:44:06] [DEBUG] Found extension class Database in db_refesh: Attempting to load
[16:44:06] [DEBUG] Loaded Extension: db_refesh
[16:44:06] [DEBUG] No setup function found in help
[16:44:06] [DEBUG] Found extension class Help in help: Attempting to load
[16:44:06] [DEBUG] Added callback: Help._about
[16:44:06] [DEBUG] Added callback: Help.help
[16:44:06] [DEBUG] Loaded Extension: help
[16:44:06] [DEBUG] No setup function found in channel_listen
[16:44:06] [DEBUG] Found extension class ChannelListen in channel_listen: Attempting to load
[16:44:06] [DEBUG] Added callback: ChannelListen._setup
[16:44:06] [DEBUG] Loaded Extension: channel_listen
[16:44:06] [DEBUG] Added callback: on_start
[16:44:06] [DEBUG] 1 callbacks have been loaded from __main__.
[16:44:06] [DEBUG] Added callback: _disconnect
[16:44:06] [DEBUG] Added callback: _on_websocket_ready
[16:44:06] [DEBUG] Added callback: on_autocomplete_completion
[16:44:06] [DEBUG] Added callback: on_autocomplete_error
[16:44:06] [DEBUG] Added callback: on_command_completion
[16:44:06] [DEBUG] Added callback: on_command_error
[16:44:06] [DEBUG] Added callback: on_component_completion
[16:44:06] [DEBUG] Added callback: on_component_error
[16:44:06] [DEBUG] Added callback: on_error
[16:44:06] [DEBUG] Added callback: on_modal_completion
[16:44:06] [DEBUG] Added callback: on_modal_error
[16:44:06] [DEBUG] Added callback: on_resume
[16:44:06] [DEBUG] 12 callbacks have been loaded from Client.
[16:44:06] [DEBUG] Attempting to login
[16:44:07] [DEBUG] Caching ingested rate limit data for: 78bb8553d9352a5a2f89f9def401287a
[16:44:07] [DEBUG] GET /users/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:44:07] [DEBUG] GET /users/@me Received 200 :: [0/1000 calls remaining]
[16:44:07] [DEBUG] Caching ingested rate limit data for: d28a133af187e91d71d96a223467ce49
[16:44:07] [DEBUG] GET /oauth2/applications/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:44:07] [DEBUG] GET /oauth2/applications/@me Received 200 :: [0/1000 calls remaining]
[16:44:07] [WARNING] Setting sync_ext to True by default for syncing changes
[16:44:07] [INFO] Loading jurigged
[16:44:07] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\Main.py
[16:44:07] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\play.py
[16:44:07] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\embed.py
[16:44:07] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\askbard.py
[16:44:07] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\noi_chu.py
[16:44:07] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\db_refesh.py
[16:44:07] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\help.py
[16:44:07] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\channel_listen.py
[16:44:07] [DEBUG] Caching ingested rate limit data for: 0a6d458104b103c7be293cc9b284d9ee
[16:44:07] [DEBUG] GET /gateway Received 200 :: [99/100 calls remaining]
[16:44:07] [INFO] Shard ID 0 | Starting Shard
[16:44:07] [INFO] Shard ID 0 | Shard is attempting to connect to gateway...
[16:44:08] [DEBUG] Shard ID 0 | Identification payload sent to gateway, requesting intents: 3407613
[16:44:08] [INFO] Shard ID 0 | Gateway connection established
[16:44:08] [DEBUG] Shard ID 0 | Session ID: 11ea95a4143b2eac3e3f960d5eae737b Trace: ['["gateway-prd-us-east1-b-cjn8",{"micros":58009,"calls":["id_created",{"micros":1081,"calls":[]},"session_lookup_time",{"micros":354,"calls":[]},"session_lookup_finished",{"micros":13,"calls":[]},"discord-sessions-prd-2-214",{"micros":55889,"calls":["start_session",{"micros":46412,"calls":["discord-api-6c869fb97d-8pwwz",{"micros":40603,"calls":["get_user",{"micros":8660},"get_guilds",{"micros":4337},"send_scheduled_deletion_message",{"micros":12},"guild_join_requests",{"micros":2},"authorized_ip_coro",{"micros":12}]}]},"starting_guild_connect",{"micros":81,"calls":[]},"presence_started",{"micros":279,"calls":[]},"guilds_started",{"micros":212,"calls":[]},"guilds_connect",{"micros":2,"calls":[]},"presence_connect",{"micros":8878,"calls":[]},"connect_finished",{"micros":8885,"calls":[]},"build_ready",{"micros":17,"calls":[]},"clean_ready",{"micros":1,"calls":[]},"optimize_ready",{"micros":0,"calls":[]},"split_ready",{"micros":1,"calls":[]}]}]}]']
[16:44:08] [DEBUG] Dispatching Event: websocket_ready
[16:44:08] [DEBUG] Dispatching Event: guild_join
[16:44:08] [DEBUG] Dispatching Event: guild_join
[16:44:08] [DEBUG] Dispatching Event: guild_join
[16:44:08] [DEBUG] Dispatching Event: guild_join
[16:44:08] [DEBUG] Dispatching Event: guild_join
[16:44:09] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[16:44:09] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:44:09] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[16:44:09] [DEBUG] Waiting for bucket dd84d9818020c27975c0d5ab4a794d44 to unlock.
[16:44:09] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[16:44:09] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:44:09] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[16:44:09] [DEBUG] 0 is already up-to-date with 14 commands.
[16:44:09] [DEBUG] Sync of 1 scopes took 0.8408874000015203 seconds
[16:44:09] [DEBUG] Dispatching Event: startup
[16:44:09] [DEBUG] Sending data to websocket: {"op": 3, "d": {"since": 1703756649620, "activities": [{"name": "/help for helpful", "type": 0}], "status": "idle", "afk": false}}
[16:44:11] [DEBUG] 1174695146214928500::play should be called
[16:44:11] [DEBUG] Dispatching Event: message_create
[16:44:11] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[16:44:11] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4OTg2NjM4MDY4Njc5MDcxNzp6YVdFVzZOYWhwSEpCMHY3TzN2U2dlT3ZtNDl0Rm83RHhJMHRtV3pUZW5zS0c5dktQQmp2T0hUNFJLZHFKc2pVOEJSanBhcWdzdmE5Z0l4OHJNcllicmg0WHFTckZ0RnRSZ3NqeUMySnk2ZW0xcVRrdzhEU2U3NGFYZmlmR3NneQ/callback Received 204 :: [4/5 calls remaining]
[16:44:11] [DEBUG] Sending data to websocket: {"op": 4, "d": {"guild_id": 1067024293709496340, "channel_id": 1189851990356918383, "self_mute": false, "self_deaf": false}}
[16:44:11] [DEBUG] Waiting for voice connection data...
[16:44:11] [DEBUG] Dispatching Event: voice_user_join
[16:44:12] [DEBUG] Attempting to initialise voice gateway...
[16:44:12] [DEBUG] Voice Connection has identified itself to Voice Gateway
[16:44:12] [DEBUG] Discord send VC Ready! Establishing a socket connection...
[16:44:12] [DEBUG] IP Discovery in progress...
[16:44:12] [DEBUG] Voice Initial Response Received: b'\x00\x02\x00F\x00\x05cY116.100.183.123\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xdfU'
[16:44:12] [DEBUG] IP Discovered: 116.100.183.123 #57173
[16:44:12] [DEBUG] Sending data to websocket: {"op": 1, "d": {"protocol": "udp", "data": {"address": "116.100.183.123", "port": 57173, "mode": "xsalsa20_poly1305_lite"}}}
[16:44:12] [DEBUG] Starting UDP Keep Alive
[16:44:12] [DEBUG] Unhandled OPCODE: 18 = data = {'user_id': '610841389164396565', 'flags': 2}
[16:44:12] [DEBUG] Unhandled OPCODE: 20 = data = {'user_id': '610841389164396565', 'platform': 0}
[16:44:12] [INFO] Voice connection established; using xsalsa20_poly1305_lite
[16:44:13] [DEBUG] Dispatching Event: command_error
[16:44:13] [DEBUG] Dispatching Event: command_completion
[16:44:13] [DEBUG] Dispatching Event: error
[16:44:13] [INFO] Command Called: play with event.ctx.args = ['https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'] | event.ctx.kwargs = {'song': 'https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'}
[16:44:13] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 129, in play
    await music_queues.playlist_process(url_list)
  File "D:\Project\Python\qpneruy-Git\modules\Music_queue.py", line 39, in playlist_process
    audio = await youtube_dl.get_audio(song)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\modules\YT_Downloader.py", line 53, in get_audio
    data = await asyncio.to_thread(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\asyncio\threads.py", line 25, in to_thread
    return await loop.run_in_executor(None, func_call)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\modules\YT_Downloader.py", line 54, in <lambda>
    lambda: cfg_video.extract_info(url, download=False)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\yt_dlp\YoutubeDL.py", line 1556, in extract_info
    if not ie.suitable(url):
           ^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\yt_dlp\extractor\lazy_extractors.py", line 152, in suitable
    qs = parse_qs(url)
         ^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\yt_dlp\utils\_utils.py", line 2484, in parse_qs
    return urllib.parse.parse_qs(urllib.parse.urlparse(url).query, **kwargs)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\urllib\parse.py", line 394, in urlparse
    url, scheme, _coerce_result = _coerce_args(url, scheme)
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\urllib\parse.py", line 133, in _coerce_args
    return _decode_args(args) + (_encode_result,)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\urllib\parse.py", line 117, in _decode_args
    return tuple(x.decode(encoding, errors) if x else '' for x in args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\urllib\parse.py", line 117, in <genexpr>
    return tuple(x.decode(encoding, errors) if x else '' for x in args)
                 ^^^^^^^^
AttributeError: 'coroutine' object has no attribute 'decode'

[16:44:15] [DEBUG] Sending heartbeat every 13.75 seconds
[16:44:15] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.12218898340737172}
[16:44:15] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:44:15] [DEBUG] ❤ Heartbeat acknowledged after 0.07997 seconds
[16:44:23] [DEBUG] Sending heartbeat every 41.25 seconds
[16:44:23] [DEBUG] Sending data to websocket: {"op": 1, "d": 10}
[16:44:23] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:44:23] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:44:29] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.3027922157029108}
[16:44:29] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:44:29] [DEBUG] ❤ Heartbeat acknowledged after 0.08081 seconds
[16:44:37] [DEBUG] Caching play
[16:44:37] [DEBUG] Update play.Music.play:94
[16:44:37] [DEBUG] Update play.Music.play:94
[16:44:37] [DEBUG] Update play.Music:70
[16:44:37] [DEBUG] Update play.Music:70
[16:44:37] [DEBUG] Update play:1
[16:44:37] [DEBUG] Update play:1
[16:44:37] [DEBUG] Checking play
[16:44:37] [DEBUG] Checking for changes
[16:44:37] [DEBUG] No changes detected
[16:44:39] [DEBUG] Dispatching Event: disconnect
[16:44:39] [DEBUG] Stopping the bot.
[16:44:39] [INFO] Shard ID 0 | Stopping Shard
[16:44:48] [DEBUG] Running client sanity checks...
[16:44:48] [DEBUG] Loaded Extension: interactions.ext.jurigged
[16:44:49] [DEBUG] No setup function found in play
[16:44:49] [DEBUG] Found extension class Music in play: Attempting to load
[16:44:49] [DEBUG] Added callback: Music._menu
[16:44:49] [DEBUG] Added callback: Music._pause
[16:44:49] [DEBUG] Added callback: Music._resume
[16:44:49] [DEBUG] Added callback: Music._skip
[16:44:49] [DEBUG] Added callback: Music._stop
[16:44:49] [DEBUG] Added callback: Music.play
[16:44:49] [DEBUG] Loaded Extension: play
[16:44:50] [DEBUG] No setup function found in askbard
[16:44:50] [DEBUG] Found extension class Bard in askbard: Attempting to load
[16:44:50] [DEBUG] Added callback: Bard._askbard
[16:44:50] [DEBUG] Added callback: Bard._endbard
[16:44:50] [DEBUG] Added callback: Bard.set_bard_channel
[16:44:50] [DEBUG] Loaded Extension: askbard
[16:44:50] [DEBUG] No setup function found in noi_chu
[16:44:50] [DEBUG] Found extension class NoiChu in noi_chu: Attempting to load
[16:44:50] [DEBUG] Added callback: NoiChu._reset
[16:44:50] [DEBUG] Added callback: NoiChu._world_setup
[16:44:50] [DEBUG] Loaded Extension: noi_chu
[16:44:50] [DEBUG] No setup function found in db_refesh
[16:44:50] [DEBUG] Found extension class Database in db_refesh: Attempting to load
[16:44:50] [DEBUG] Loaded Extension: db_refesh
[16:44:50] [DEBUG] No setup function found in help
[16:44:50] [DEBUG] Found extension class Help in help: Attempting to load
[16:44:50] [DEBUG] Added callback: Help._about
[16:44:50] [DEBUG] Added callback: Help.help
[16:44:50] [DEBUG] Loaded Extension: help
[16:44:50] [DEBUG] No setup function found in channel_listen
[16:44:50] [DEBUG] Found extension class ChannelListen in channel_listen: Attempting to load
[16:44:50] [DEBUG] Added callback: ChannelListen._setup
[16:44:50] [DEBUG] Loaded Extension: channel_listen
[16:44:50] [DEBUG] Added callback: on_start
[16:44:50] [DEBUG] 1 callbacks have been loaded from __main__.
[16:44:50] [DEBUG] Added callback: _disconnect
[16:44:50] [DEBUG] Added callback: _on_websocket_ready
[16:44:50] [DEBUG] Added callback: on_autocomplete_completion
[16:44:50] [DEBUG] Added callback: on_autocomplete_error
[16:44:50] [DEBUG] Added callback: on_command_completion
[16:44:50] [DEBUG] Added callback: on_command_error
[16:44:50] [DEBUG] Added callback: on_component_completion
[16:44:50] [DEBUG] Added callback: on_component_error
[16:44:50] [DEBUG] Added callback: on_error
[16:44:50] [DEBUG] Added callback: on_modal_completion
[16:44:50] [DEBUG] Added callback: on_modal_error
[16:44:50] [DEBUG] Added callback: on_resume
[16:44:50] [DEBUG] 12 callbacks have been loaded from Client.
[16:44:50] [DEBUG] Attempting to login
[16:44:50] [DEBUG] Caching ingested rate limit data for: 78bb8553d9352a5a2f89f9def401287a
[16:44:50] [DEBUG] GET /users/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:44:50] [DEBUG] GET /users/@me Received 200 :: [0/1000 calls remaining]
[16:44:51] [DEBUG] Caching ingested rate limit data for: d28a133af187e91d71d96a223467ce49
[16:44:51] [DEBUG] GET /oauth2/applications/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:44:51] [DEBUG] GET /oauth2/applications/@me Received 200 :: [0/1000 calls remaining]
[16:44:51] [WARNING] Setting sync_ext to True by default for syncing changes
[16:44:51] [INFO] Loading jurigged
[16:44:51] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\Main.py
[16:44:51] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\play.py
[16:44:51] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\embed.py
[16:44:51] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\askbard.py
[16:44:51] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\noi_chu.py
[16:44:51] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\db_refesh.py
[16:44:51] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\help.py
[16:44:51] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\channel_listen.py
[16:44:51] [DEBUG] Caching ingested rate limit data for: 0a6d458104b103c7be293cc9b284d9ee
[16:44:51] [DEBUG] GET /gateway Received 200 :: [99/100 calls remaining]
[16:44:51] [INFO] Shard ID 0 | Starting Shard
[16:44:51] [INFO] Shard ID 0 | Shard is attempting to connect to gateway...
[16:44:51] [DEBUG] Shard ID 0 | Identification payload sent to gateway, requesting intents: 3407613
[16:44:52] [INFO] Shard ID 0 | Gateway connection established
[16:44:52] [DEBUG] Shard ID 0 | Session ID: 6fa1a71a8d467bbd6ac7ab10f07bc673 Trace: ['["gateway-prd-us-east1-d-df49",{"micros":145484,"calls":["id_created",{"micros":662,"calls":[]},"session_lookup_time",{"micros":5571,"calls":[]},"session_lookup_finished",{"micros":16,"calls":[]},"discord-sessions-prd-2-51",{"micros":137872,"calls":["start_session",{"micros":118042,"calls":["discord-api-6c869fb97d-kqmjn",{"micros":110234,"calls":["get_user",{"micros":12806},"get_guilds",{"micros":11888},"send_scheduled_deletion_message",{"micros":10},"guild_join_requests",{"micros":16344},"authorized_ip_coro",{"micros":17}]}]},"starting_guild_connect",{"micros":66,"calls":[]},"presence_started",{"micros":491,"calls":[]},"guilds_started",{"micros":213,"calls":[]},"guilds_connect",{"micros":2,"calls":[]},"presence_connect",{"micros":19027,"calls":[]},"connect_finished",{"micros":19033,"calls":[]},"build_ready",{"micros":23,"calls":[]},"clean_ready",{"micros":1,"calls":[]},"optimize_ready",{"micros":1,"calls":[]},"split_ready",{"micros":0,"calls":[]}]}]}]']
[16:44:52] [DEBUG] Dispatching Event: websocket_ready
[16:44:52] [DEBUG] Dispatching Event: guild_join
[16:44:52] [DEBUG] Dispatching Event: guild_join
[16:44:52] [DEBUG] Dispatching Event: guild_join
[16:44:52] [DEBUG] Dispatching Event: guild_join
[16:44:52] [DEBUG] Dispatching Event: guild_join
[16:44:52] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[16:44:52] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:44:52] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[16:44:52] [DEBUG] Waiting for bucket dd84d9818020c27975c0d5ab4a794d44 to unlock.
[16:44:53] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[16:44:53] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:44:53] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[16:44:53] [DEBUG] 0 is already up-to-date with 14 commands.
[16:44:53] [DEBUG] Sync of 1 scopes took 0.7477758000022732 seconds
[16:44:53] [DEBUG] Dispatching Event: startup
[16:44:53] [DEBUG] Sending data to websocket: {"op": 3, "d": {"since": 1703756693055, "activities": [{"name": "/help for helpful", "type": 0}], "status": "idle", "afk": false}}
[16:44:54] [DEBUG] 1174695146214928500::play should be called
[16:44:55] [DEBUG] Dispatching Event: message_create
[16:44:55] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[16:44:55] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4OTg2NjU2Mzc4OTEzMTkwNzpQUDUyTEE3VW9YZWFBUEtWamk4aGY5dUtEbmlRMTF1UXA5dFVoVzNTdlBsMTVOcVZNMU1HWVhrVmhROUR2WWNnVW84WHo3ZmNnSzhhejFtT05jWjM5WkpjNUxtaTdQdzRBMVBxSk1LeFB4SlNkSnJOb21xbHZZVlAxNW55QWhhbw/callback Received 204 :: [4/5 calls remaining]
[16:44:55] [DEBUG] Sending data to websocket: {"op": 4, "d": {"guild_id": 1067024293709496340, "channel_id": 1189851990356918383, "self_mute": false, "self_deaf": false}}
[16:44:55] [DEBUG] Waiting for voice connection data...
[16:44:55] [DEBUG] Dispatching Event: voice_user_join
[16:44:55] [DEBUG] Attempting to initialise voice gateway...
[16:44:56] [DEBUG] Voice Connection has identified itself to Voice Gateway
[16:44:56] [DEBUG] Discord send VC Ready! Establishing a socket connection...
[16:44:56] [DEBUG] IP Discovery in progress...
[16:44:56] [DEBUG] Voice Initial Response Received: b'\x00\x02\x00F\x00\x05cb116.100.183.123\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xfc\xca'
[16:44:56] [DEBUG] IP Discovered: 116.100.183.123 #64714
[16:44:56] [DEBUG] Sending data to websocket: {"op": 1, "d": {"protocol": "udp", "data": {"address": "116.100.183.123", "port": 64714, "mode": "xsalsa20_poly1305_lite"}}}
[16:44:56] [DEBUG] Starting UDP Keep Alive
[16:44:56] [DEBUG] Unhandled OPCODE: 18 = data = {'user_id': '610841389164396565', 'flags': 2}
[16:44:56] [DEBUG] Unhandled OPCODE: 20 = data = {'user_id': '610841389164396565', 'platform': 0}
[16:44:56] [INFO] Voice connection established; using xsalsa20_poly1305_lite
[16:44:57] [DEBUG] Dispatching Event: command_error
[16:44:57] [DEBUG] Dispatching Event: command_completion
[16:44:57] [DEBUG] Dispatching Event: error
[16:44:57] [INFO] Command Called: play with event.ctx.args = ['https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'] | event.ctx.kwargs = {'song': 'https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'}
[16:44:57] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 125, in play
    data = await cfg_playlist.extract_info(song, download=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: object dict can't be used in 'await' expression

[16:45:00] [DEBUG] Sending heartbeat every 41.25 seconds
[16:45:00] [DEBUG] Sending data to websocket: {"op": 1, "d": 10}
[16:45:00] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:45:01] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:45:01] [DEBUG] Sending heartbeat every 13.75 seconds
[16:45:01] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.18651802496179604}
[16:45:01] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:45:01] [DEBUG] ❤ Heartbeat acknowledged after 0.09099 seconds
[16:45:03] [DEBUG] Caching noi_chu
[16:45:03] [DEBUG] Checking noi_chu
[16:45:03] [DEBUG] Checking for changes
[16:45:03] [DEBUG] No changes detected
[16:45:15] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.4442891967521514}
[16:45:15] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:45:15] [DEBUG] ❤ Heartbeat acknowledged after 0.08372 seconds
[16:45:28] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.4038912889548032}
[16:45:28] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:45:28] [DEBUG] ❤ Heartbeat acknowledged after 0.08323 seconds
[16:45:42] [DEBUG] Sending data to websocket: {"op": 1, "d": 10}
[16:45:42] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:45:42] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:45:42] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.5083169444337177}
[16:45:42] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:45:42] [DEBUG] ❤ Heartbeat acknowledged after 0.08483 seconds
[16:45:56] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.29196523948272857}
[16:45:56] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:45:56] [DEBUG] ❤ Heartbeat acknowledged after 0.08629 seconds
[16:45:58] [DEBUG] Caching play
[16:45:58] [DEBUG] Run play:6 | import asyncio
[16:45:58] [DEBUG] Update play.Music.play:94
[16:45:58] [DEBUG] Run play:6 | import asyncio
[16:45:58] [DEBUG] Update play.Music:70
[16:45:58] [DEBUG] Update play.Music.play:94
[16:45:58] [DEBUG] Update play:1
[16:45:58] [DEBUG] Update play.Music:70
[16:45:58] [DEBUG] Update play:1
[16:45:58] [DEBUG] Checking play
[16:45:58] [DEBUG] Checking for changes
[16:45:58] [DEBUG] No changes detected
[16:46:06] [DEBUG] Caching play
[16:46:06] [DEBUG] Update play.Music.play:94
[16:46:06] [DEBUG] Update play.Music:70
[16:46:06] [DEBUG] Update play:1
[16:46:06] [DEBUG] Checking play
[16:46:06] [DEBUG] Checking for changes
[16:46:06] [DEBUG] No changes detected
[16:46:08] [DEBUG] Dispatching Event: disconnect
[16:46:08] [DEBUG] Stopping the bot.
[16:46:08] [INFO] Shard ID 0 | Stopping Shard
